etc/config/java.defaults.properties

```diff
@@ -0,0 +1,17 @@
+# Default settings for Java
+compilers=&java
+compilerType=java
+versionFlag=-version
+objdumper=javap
+
+group.java.compilers=javacdefault
+compiler.javacdefault.exe=/usr/bin/javac
+compiler.javacdefault.name=javac default
+
+defaultCompiler=javacdefault
+demangler=
+postProcess=
+options=
+supportsBinary=false
+needsMulti=false
+supportsExecute=false
```

Comment:
Can you please ensure that there's a newline at EOL? :)

Reply:
Done :)